### ISSUE TITLE
A11Y: SVG of notched-outline should have focusable="false" attribute for IE11

### ISSUE THREAD

**Issue by mathiasschopmans** [material-components/material-components-web#3582]
### What MDC Web Version are you using?
- Version: latest / 0.39.x
- Components: textfield with outline

### What browser(s) is this bug affecting?
Internet Explorer 11

### What OS are you using?
Windows 7 (modern.ie virtual box virtual machine)

### What are the steps to reproduce the bug?
The bug can be reproduced, when viewing the docs with the mentioned browser.
- visit https://material-components.github.io/material-components-web-catalog/#/component/text-field
- click inside the first outlined textfield (Standard)
- press "tab"

### What is the expected behavior?
- The SVG element shouldn't be focusable
- Focus should move to the next visible input-field (Dense)

### What is the actual behavior?
- In IE11 the SVG will gain the focus therefore you need to
- press "tab" again to move the focus to the second element

### Any other information you believe would be useful?
This can be fixed via the `focusable="false"` attribute:

```html
<svg focusable="false">
    <path class="mdc-notched-outline__path" d=""/>
</svg>
```

If this issue will be accpeted, there README.mds of textfield and notched outline needs to be adapted and examples needs to be updated.

**Comment by kfranqueiro** [comment]
This will be resolved with #4035, because notched outline will no longer use SVG.

**Comment by mathiasschopmans** [comment]
Awesome news.
